src/test/data/XmlSerializableAddressBookTest/typicalPersonsAddressBook.xml

```diff
@@ -85,4 +85,82 @@
         <meeting>0000000000</meeting>
         <picture>/images/placeholder_image.jpg</picture>
```

Comment:
It's not modified here, which causes your XML Serializable address book test to fail.

Reply:
Openheimer also has error; phone number is mismatched.

Reply:
OpenHeimer's address is mismatched as well.